### DISCUSSION TITLE
Create iModel- Local file viewed versions

### DISCUSSION THREAD

**Discussion by sampath-agathsya** [iTwin/itwinjs-core#4712] [Q&A]
1) when we upload local file during creating iModel it is showing only white mesh model. I want to show textured 3d model.
2) how to show that textured 3Dmodel in imodelhub and how to create Named Version during imodel creation?
  
but 4 months ago when i upload .imodel file it is showing every thing like textured 3D and we can see those 3D model in imodelhub when you click on view tab. Now It doesn't show any thing in imodelHub. 

NOW I uploaded .dgn file format.
whenever you create named versions manually in iTwin platform then only it is visible in imodelhub with white color 
![Screenshot 2022-11-22 130258](https://user-images.githubusercontent.com/96165248/203252638-4ec58a98-9218-4da3-9041-6583c046c03e.png)
mesh only . why?

**Comment by arnab-git-hub** [comment]
@sampath-agathsya Is it happening for one particular dgn file/imodel or all of your imodels? Can you please provide a sample dgn file?

**Reply by arnab-git-hub** [reply]
@sampath-agathsya Following up as we didn't hear back from you. If the issue still persists, please let us know.
